rrdd/xcp_rrdd.ml

```diff
@@ -683,21 +683,26 @@ let doc = String.concat "\n" [
     "This service maintains a list of registered datasources (shared memory pages containing metadata and time-varying values), periodically polls the datasources and records historical data in RRD format.";
   ]
 
-(** write memory stats to the filesystem so they can be propagated to xapi *)
+(** write memory stats to the filesystem so they can be propagated to xapi
+ * The structure contains the name and the number of pages to dedicate to rrds
+ * - mem_vms: ~1024 VMs supported * 1024 bytes per VM * (4096 bytes per page) = 256 pages
+ *   The json-like serialization for the 3 dss in dss_mem_vms takes 622 bytes.
+ *   These bytes plus some overhead make 1024 bytes an upper bound.
+ *)
 let stats_to_write = [
-  "mem_host";
-  "mem_vms"
+  "mem_host", 1;
+  "mem_vms", 256
```

Comment:
might be better to use a constant and calculate the value in code, so if we want to increase this in the future it is easy to find, the only thing that won't change is the pagesize:
```
let max_supported_rrd_vms = 1024
let vm_json_metadata_max = 1024
let max_rrd_pages = (max_supported_rrd_vms * vm_json_metadata_max + 4095) / 4096
```

Reply:
That's a good idea, I also prefer to be explicit than implicit.